src/opening-period/page/CreateNewOpeningPeriodPage.scss

```diff
@@ -0,0 +1,122 @@
+@import 'node_modules/hds-design-tokens/lib/all.scss';
+
+.add-new-opening-period-page-header-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.add-new-opening-period-page-title {
+  margin-top: $spacing-2-xs;
+  font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
+  font-size: $fontsize-heading-m;
+  color: var(--color-coat-of-arms-blue);
+}
+
+.add-new-opening-period-form {
+  label {
+    display: block;
+    margin-top: $spacing-m;
+  }
+
+  input {
+    display: block;
+  }
+
+  h3 {
+    margin-bottom: 0;
+  }
+
+  margin-top: $spacing-xl;
+}
+
+.opening-period-optional-description {
+  padding: $spacing-layout-2-xs;
+  width: calc(0.5808333 * 100%);
+  height: 164px;
+  border: 2px solid rgb(128, 128, 128);
+  margin-bottom: $spacing-layout-2-xs;
+
+  @media screen and (max-width: $breakpoint-m) {
+    width: 100%;
+  }
+}
+
+.opening-period-section-title {
+  font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
+  font-size: $fontsize-heading-xxs;
+}
+
+.add-new-opening-period-title {
+  font-family: HelsinkiGrotesk-Regular, Arial, sans-serif;
+  font-size: $fontsize-body-l;
+  line-height: 2.5;
+  padding-left: 16px;
+  padding-right: 16px;
+  border: 2px solid rgb(128, 128, 128);
+  min-width: 400px;
+}
+
+.opening-period-time-period {
+  display: flex;
+  align-items: center;
+}
+
+.dash-between-begin-and-end-date {
+  margin-bottom: -35px;
+  margin-left: 10px;
+  margin-right: 10px;
+  font-family: HelsinkiGrotesk-Bold, Arial, sans-serif;
+}
+
+.form-content-container {
+  padding-left: $spacing-layout-xs;
+  padding-right: $spacing-layout-xs;
+}
+
+.add-new-opening-period-final-action-row-container {
+  margin-top: 116px;
+}
+
+.add-new-opening-period-final-action-button.add-new-opening-period-final-action-button {
+  font-family: HelsinkiGrotesk-Medium, Arial, sans-serif;
+  margin-right: $spacing-m;
+  border-color: var(--color-coat-of-arms-blue);
+}
+
+.publish-new-opening-period-button.publish-new-opening-period-button {
+  color: var(--add-new-opening-period-button-color);
+  background-color: var(--color-coat-of-arms-blue);
+
+  &:hover {
+    background-color: var(--color-coat-of-arms-blue-dark-50);
+    border-color: var(--color-coat-of-arms-blue-dark-50);
+  }
+
+  &:focus {
+    background-color: var(--color-coat-of-arms-blue);
+    border-color: var(--color-coat-of-arms-blue) !important;
+    outline-color: var(--color-coat-of-arms-blue);
+  }
+}
+
+.cancel-creation-of-new-opening-period-button.cancel-creation-of-new-opening-period-button {
+  color: var(--color-coat-of-arms-blue);
+
+  &:hover {
+    background-color: $color-coat-of-arms-blue-light-5;
+    border-color: var(--color-coat-of-arms-blue);
+  }
+
+  &:focus {
+    border-color: var(--color-coat-of-arms-blue) !important;
+    outline-color: var(--color-coat-of-arms-blue);
+    color: var(--color-coat-of-arms-blue) !important;
+  }
+}
+
+.disabled-publish-button.disabled-publish-button {
```

Comment:
It seems I've actually forgot to implement this disabled button to the component. 😅  Will add it to this pr:

- [ ] Implement disabled button